src/vellum/workflows/nodes/displayable/bases/prompt_deployment_node.py

```diff
@@ -76,6 +87,49 @@ def _get_prompt_event_stream(self) -> Iterator[ExecutePromptEvent]:
             request_options=request_options,
         )
 
+    def _process_prompt_event_stream(self) -> Generator[BaseOutput, None, Optional[List[PromptOutput]]]:
+        """Override the base prompt node _process_prompt_event_stream()"""
+        self._validate()
+        try:
+            prompt_event_stream = self._get_prompt_event_stream()
+        except ApiError:
+            if self.ml_model_fallbacks:
```

Comment:
[nit] let's make sure that the ApiError code is less than 500, otherwise that's an us error that could be wasting their tokens